packages/react-components/src/components/VideoTile.tsx

```diff
@@ -151,7 +163,47 @@ const DefaultPlaceholder = (props: CustomAvatarOptions): JSX.Element => {
   );
 };
 
+/* @conditional-compile-remove(pinned-participants) */
```

Comment:
You don't need to conditionally remove this interface since it's being exported and won't create warnings.

Reply:
done